docs-client/src/components/VariableList/index.tsx

```diff
@@ -83,168 +90,136 @@ export default function({
   );
 }
 
-interface State {
-  isEmpty: boolean;
-  isBeans: boolean[];
-  openBeans: boolean[];
-  colSpanLength: number;
-  childStates: { [key: number]: State };
-}
-
 interface OwnProps {
   indent: number;
-  hasBean: boolean;
-  state?: State;
-  setChildStateInParent?: (index: number, childState: State) => void;
-  childIndex: number;
+  hasLocation: boolean;
+  hidden?: boolean;
 }
 
 type FieldInfosProps = OwnProps & Props;
 
-class FieldInfos extends React.Component<FieldInfosProps, State> {
-  private static generateKey(pre: string): string {
-    return `${pre}_${new Date().getTime()}`;
-  }
+interface FieldInfoProps {
+  hasLocation: boolean;
+  hidden?: boolean;
+  indent: number;
+  variable: Variable;
+  specification: Specification;
+  title: string;
+}
 
-  private static indentString(indent: number, s: string): string {
-    return `${'\xa0'.repeat(indent)}${s}`;
-  }
+const indentString = (indent: number, s: string): string => {
```

Comment:
Question: Why are we not using `function` here?

Reply:
I've noticed most code using lambdas everywhere now. Since we don't support old browsers anymore there's no performance difference (not transpiled) and I guess it's nice to just not think about whether it's safe to use function or not and consistently use the safer lambda

Reply:
Thanks for the answer.